packages/test/test-end-to-end-tests/src/test/gc/gcSweepAttachmentBlobs.spec.ts

```diff
@@ -64,9 +64,16 @@ describeNoCompat("GC attachment blob sweep tests", (getTestObjectProvider) => {
 		container: IContainer,
 		blobNodePath: string,
 		messagePrefix: string,
+		isSummarizerContainer = false,
 	) {
 		const blobId = blobNodePath.split("/")[2];
-		const response = await container.request({ url: blobNodePath });
+		const entryPoint = (await container.getEntryPoint()) as ITestDataObject;
+		const runtime = isSummarizerContainer
+			? ((entryPoint as any).runtime as IContainerRuntimeBase)
+			: entryPoint._context.containerRuntime;
+		const response = await (runtime as any).resolveHandle({
```

Comment:
Example of `(containerRuntime as any).resolveHandle(...)`

Reply:
Didn't we add a deprecated interface that has resolveHandle?

Reply:
You're correct. I actually changed these calls to be `(containerRuntime as ContainerRuntime).resolveHandle` so the tests don't break once the methods are removed from interfaces.
Not sure if it's worth revisiting the tests later to make them work without the `ContainerRuntime.resolveHandle` method.